common/src/main/generated/data/bingo/tags/entity_type/hostile.json

```diff
@@ -4,12 +4,14 @@
     "minecraft:bogged",
     "minecraft:breeze",
     "minecraft:cave_spider",
+    "minecraft:creaking",
+    "minecraft:creaking_transient",
```

Comment:
Again, should exclude the experiments, I don't think this tag will load without it enabled

Reply:
Maybe exclude disabled entries from cycle icons if they're not already?

Reply:
That's a good idea.